inst/ms/aggreCAT.qmd

```diff
@@ -400,39 +423,43 @@ for a single claim to illustrate this structure of the core
 `data_ratings` dataset.
 
 ```{r}
-#| label: data_ratings
+#| label: data_ratings-sample
 #| message: false
 #| prompt: true
+#| results: hold
+
 library(tidyverse,quietly = TRUE)
 library(aggreCAT)
 data(data_ratings)
 data_ratings %>% 
-  print(n = 18)
+  dplyr::filter(paper_id == dplyr::first(paper_id), 
+                user_name == dplyr::first(user_name)) %>% 
+  print(., n = nrow(.))
 ```
 
 Not all data necessary for constructing weights on performance is
 contained in `data_ratings`. Additional data collected as part of the
 repliCATS IDEA protocol are contained within separate datasets to
-`data_ratings`. Justifications for giving particular judgements are
-contained in `data_justifications`. on the repliCATS platform users were
-given the option to comment on others' justifications (`data_comments`),
-to vote on others' comments (`data_comment_ratings`) and on others'
-justifications (`data_justification_ratings`). Finally, [aggreCAT]{.pkg}
-contains three 'supplementary' datasets containing data collected
-externally to the repliCATS IDEA protocol: `data_supp_quiz`,
+`data_ratings`. On the repliCATS platform users were given the option to
+comment on others' justifications (`data_comments`), to vote on others'
+comments (`data_comment_ratings`) and on others' justifications
+(`data_justification_ratings`). Justifications for giving particular
+judgements are contained in `data_justifications`. Finally,
```

Comment:
I'd list and explain `data_justifications` before the other three, especially since one refers to it